01-landing-page/scss/style.scss

```diff
@@ -0,0 +1,294 @@
+    background-color: $green;
+    color: white;
+    font-size: .9rem;
+    display: inline-block;
+    width: 10rem;
+    padding: 1rem 1.5rem;
+    border: none;
+    &:hover {
+        background-color: lighten($color: $green, $amount: 5);
+    }
+}
+.img-container {
+    height: 60vh;
+}
+.hero__img {
+    grid-area: img;
+    width: 100%;
+    height: auto;
+}
+
+
+@media screen and (max-width: 790px) {
+
+ //Navbar
+
+    .nav {
+        position: relative;
+        left: 0;
+        top: 0;
+        z-index: 1;
+    }
+
+
+    .nav__list {
+        width: 100%;
+        height: 100vh;
+        background-color:$light-gray;
+        position: absolute;
+        left: 0%;
+        top: 0;
+        transform: translateX(-100%);
+        transition: transform .5s ease-in;
+        z-index: 250;
+
+        .d-flex {
+            flex-direction: column;
+            justify-content: space-evenly;
+            height: 100vh;
+        }
+    }
+
+    .nav__list__item {
+        margin-left: 0;
+        .nav__link {
+            color: $dark-blue;
+            font-size: 1.3rem;
+        }
+    }
+
+
+    .menu__toggler {
+        position: relative;
+        display: block;
+        cursor: pointer;
+        height: 2rem;
+        width: 2rem;   
+        z-index: 300;
+    }
+
+    .menu__toggler  .menu__toggler__icon {
+        display: block;
+        height: 2px;
+        width: 2rem;
+        background-color: $dark-blue;
+        position: relative;
+        top: 1rem;
+        z-index: 350;
+        border: none;
+        cursor: pointer;
+        transition: background-color .4s ease-out;
+
+        &::before  {
+            content: '';
+            position: absolute;
+            top: 8px;
+            left: 0;
+            height: 2px;
+            width: 2rem;
+            background-color: $dark-blue;
+            transition: all .4s ease-out;
+        }
+
+        &::after {
+            content: '';
+            position: absolute;
+            top: -8px;
+            left: 0;
+            height: 2px;
+            width: 2rem;
+            background-color: $dark-blue;
+            transition: all .4s ease-out;
+        }
+    }
+   
+    .open {
+        transform: translateX(0)!important;
+        overflow-y: hidden;
+    }
+
+    .nav__list.open + .menu__toggler > .menu__toggler__icon {
+        background-color: $light-gray;
+        transition: background-color .1s ease-in-out;
+
+        &::before  {
+            transform: rotate(45deg);
+            transition: all .4s ease-in-out;
+
+        }
+
+        &::after  {
+            transform: rotate(-43deg);
+            position: absolute;
+            top: 8px;
+            transition: all .4s ease-in-out;
+
+        }
+    }
+
+//Hero 
+
+    .hero {
+        height: 130vh;
+        display: grid;
+        grid-template-columns: 100%;
+        grid-template-rows: 10% auto auto;
+        grid-template-areas:
+        'heading'
+        'img'
+        'subheading';
```

Comment:
In mobile all rows are "equal" so you can just leave it `grid-template-rows: min-content`